src/components/CollectionsAndPalettes/common/CollectionsHeader/CollectionsHeader.module.scss

```diff
@@ -0,0 +1,76 @@
+@import "styles/global";
+@import "styles/variables.scss";
+
+.Header {
+  background-color: black;
+  display: flex;
+  flex-direction: row;
+  color: white;
+  padding: $margin-small-4 $margin-huge-5;
+  column-gap: $margin-medium-3;
+}
+.headerLogo {
+  h1 {
+    font-size: 26px;
+    color: white;
+  }
+  div {
+    display: flex;
+    align-items: center;
+    margin-right: 0 !important;
+    svg {
+      height: 40px;
+
+      path {
+        fill: white;
+      }
+    }
+  }
+}
+.navLinks {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  column-gap: $margin-medium;
+
+  .navLink {
+    display: flex;
+    flex-direction: row;
+    align-items: baseline;
+
+    a {
+      color: rgba(#fff, 0.8);
+      font-size: var(--font-size-lg);
+      padding-right: 10px;
+      cursor: pointer;
+      transition: color 0.5s;
+
+      &:hover {
+        color: white;
+      }
+    }
+    i {
+      font-size: var(--font-size-sm);
+    }
+  }
+}
+
+.headerIcons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+
+  i {
+    color: rgba(#fff, 0.8);
+    padding: 0 15px 0 15px;
+    font-size: var(--font-size-xl);
+    cursor: pointer;
+    transition: color 0.5s;
+
```

Comment:
We should add
```
&:first-of-type {
    padding-left: 0;
}
&:last-of-type {
    padding-right: 0;
}
```
here, as right now the icons are not aligned with the end of the page due to the padding-right of the last icon.